kitsune/community/templates/community/email/first_l10n.ltxt

```diff
@@ -0,0 +1,58 @@
+{# This is an email.  Whitespace matters! #}
```

Comment:
This is the text-based version of the html first_l10n template which at the top says it's not localized since this is sent to localizers.

Given that, we should probably not localize this one, either.